demo/frpc.toml

```diff
@@ -2,9 +2,8 @@ serverAddr = "52.77.216.244"
 serverPort = 7000
 
 [[proxies]]
-name = "gaianet"
+name = "demo.gaianet.xyz"
```

Comment:
The script will change the subdomain to something other than `demo`. Does this name stay as `demo`?

Reply:
OK